.github/workflows/scripts/config-diff-post-comment.js

```diff
@@ -0,0 +1,61 @@
+module.exports = async ({ github, context, core, configDiff }) => {
+    try {
+        // Do not create comment if there are no configuration changes
+        if (!configDiff) {
+          console.log("No changes detected. Skipping comment creation.");
+          return;
+        }
+    
+        const commentBody = `
+### Config Diff Tool Output
+
+\`\`\`diff
+
+${configDiff}
+
+\`\`\`
+  
+   
+The above configuration changes are found in the PR. Please update the relevant release documentation if necessary.
+    `;
+    
+        const { owner, repo } = context.repo;
+        const issueNumber = context.payload.pull_request.number;
+    
+        const comments = await github.paginate(
+          github.rest.issues.listComments, {
+            owner,
+            repo,
+            issue_number: issueNumber,
+            per_page: 100,
+          }
+        );
+    
+        const existingComment = comments.find(comment => comment.body.includes("### Config Diff Tool Output"));
+    
+        if (existingComment) {
+          console.log("A config diff comment already exists, deleting it...");
+          // Update the existing comment
+          await github.rest.issues.deleteComment({
+            comment_id: existingComment.id,
+            owner,
+            repo,
+          });
+        }
+    
+        console.log("Creating a new config diff comment...");
+        // Create a new comment
+        await github.rest.issues.createComment({
+          issue_number: issueNumber,
+          owner,
+          repo,
+          body: commentBody,
+        });
+    
+        // Set the status as FAILED if any configuration changes are detected
+        console.log("Configuration changes detected: ",  configDiff);
```

Comment:
You can also print some formatter [output to the job summary](https://github.com/actions/toolkit/tree/main/packages/core#populating-job-summary).

Reply:
thank you! done. Sample action summary present here: https://github.com/ceph/ceph/actions/runs/15423167839

![image](https://github.com/user-attachments/assets/5741b99a-218f-496e-892c-e5b0af134902)